pylidc/Scan.py

```diff
@@ -27,6 +27,9 @@
     configparser = ConfigParser
 
 
+ClusterError = type('ClusterError', (Exception,), {})
```

Comment:
please inherit from Exception instead of creating a type
```suggestion
class ClusterError(Exception):
    """Raised when clustering fails to group annotations"""
```